alf/algorithms/ddpg_algorithm.py

```diff
@@ -281,14 +318,39 @@ def _update_random_action(spec, noisy_action):
         if self._rollout_random_action > 0:
             nest.map_structure(_update_random_action, self._action_spec,
                                pred_step.output)
-        return pred_step
+
+        if self.need_full_rollout_state():
```

Comment:
We want the algorithm use the same ensemble_id during an entire episode. This means that it should store ensembled_id in state and use the same ensemble_id to call actor_network

Reply:
Oh yes, good point, I think that is the reason why I had to tweak the ``ddpg_algorithm_test`` to pass the toy unittest. Updated.